src/Forms/MultipleReplace.cs

```diff
@@ -193,11 +193,7 @@ private void GeneratePreview()
                             int index = newText.IndexOf(findWhat, StringComparison.OrdinalIgnoreCase);
                             while (index >= 0)
                             {
-                                if (index < newText.Length)
-                                    newText = newText.Substring(0, index) + replaceWith + newText.Substring(index + findWhat.Length);
-                                else
-                                    newText = newText.Substring(0, index) + replaceWith;
-
+                                newText = newText.Substring(0, index) + replaceWith + newText.Substring(index + findWhat.Length);

```

Comment:
I think, the intention was `if (index + findWhat.Length < newText.Length)`. But, that isn't necessary, because the potential maximum value is `newText.Length`.